### PR TITLE
Clarify issues 1558

### PR DESCRIPTION
**Pull request by andreas-junghanns**

### CODE REVIEW

File: docs/2_3_common_states.adoc [hunk 1]
```diff
@@ -880,12 +892,22 @@ Getting variables might trigger <<selectiv-computation,computations>>.
...
+
 Function <<fmi3GetDirectionalDerivative>>::
+No restrictions.
+
 Function <<fmi3GetAdjointDerivative>>::
 No restrictions.
 
 Function <<fmi3GetOutputDerivatives>>::
-Not allowed in Model Exchange.
+Not allowed in Co-Simulation and Scheduled Execution.
```
Comment (KarlWernersson) [inline docs/2_3_common_states.adoc:913]: Not allowed in Model Exchange and Scheduled Execution

File: docs/2_4_common_schema.adoc [hunk 2]
```diff
@@ -1428,8 +1427,8 @@ All variables listed in this element must have attribute `derivative` (in order
 
 |`reinit`
 |[[reinit,`reinit`]]
-Only for Model Exchange (if only a Co-Simulation FMU, this attribute must not be present.
-If both Model Exchange and a Co-Simulation FMU, this attribute is ignored for co-simulation): +
+Only for Model Exchange (if only a Co-Simulation or Scheduled Execution FMU, this attribute must not be present.
+If both Model Exchange and a Co-Simulation or Scheduled Execution FMU, this attribute is ignored for Co-Simulation and Scheduled Execution): +
```
Comment (KarlWernersson) [inline docs/2_4_common_schema.adoc:1431]: Should we reomve the requirement to not have it for co-simulation or scheduled execution, isn't it sufficient to, to stat that is it ignored for those cases?

Comment (KarlWernersson) [reply docs/2_4_common_schema.adoc:1431]: I see potential for making erroneous modelDescription.xml say that you can export as both ME and cs but choose only CS, then it is high risk that this gets there any way.

**Review by KarlWernersson — CHANGES_REQUESTED** [review]

File: docs/2_4_common_schema.adoc (hunk 2 above)
Comment (andreas-junghanns) [reply docs/2_4_common_schema.adoc]: Fixed as suggested, now:
Only used in Model Exchange, ignored for the other interface types.

File: docs/2_3_common_states.adoc (hunk 1 above)
Comment (andreas-junghanns) [reply docs/2_3_common_states.adoc]: I changed it as suggested, now consistend with "Getting Derivatives of Continuous Outputs"

**Comment by andreas-junghanns** [comment]
@TorstenBlochwitz : I followed Karl´s suggestions. Please review and approve, then I will merge.

**Review by TorstenBlochwitz — APPROVED** [review]
Much better now.

**Pull request merged**